client/modules/IDE/components/QuickAddList/QuickAddList.jsx

```diff
@@ -10,8 +10,21 @@ const Item = ({ isAdded, onSelect, name, url }) => {
   const buttonLabel = isAdded
     ? t('QuickAddList.ButtonRemoveARIA')
     : t('QuickAddList.ButtonAddToCollectionARIA');
+
+  const handleKeyDown = (event) => {
```

Comment:
i think buttons should do this natively? might not be necssary~

Reply:
oh nvm i see it's a div not a button -- i think in that case then we should just let the button within it be focusable / interactive and let the div be a container?